dist/helpers.js

```diff
@@ -54,6 +56,21 @@ var helpers = {
     return truthTest(params, bodies, context, function (left, right) {
       return left <= right;
     });
+  },
+  contextDump: function contextDump(context, params) {
+    var formattedContext = JSON.stringify(context, null, 2);
+    if (params.to === "console") {
+      console.log(formattedContext);
+    } else {
+      var frag = document.createDocumentFragment();
+      var pre = document.createElement("pre");
+      pre.appendChild(document.createTextNode(formattedContext));
+      frag.appendChild(pre);
+      return frag;
+    }
+  },
+  "debugger": function _debugger() {
```

Comment:
No, the linter doesn't care. The call stack is preserved, but you are a few functions deeper than you would like to be. Once we have compiler helpers, I would like to change this to just put a `debugger` statement directly in the compiled template. For now this works, though.